assembl/static2/js/app/components/debate/brightMirror/fictionPreview.jsx

```diff
@@ -0,0 +1,53 @@
+// @flow
+import * as React from 'react';
+import { Link } from 'react-router';
+import truncate from 'lodash/truncate';
+
+const TITLE_MAX_CHAR = 60;
+const NAME_MAX_CHAR = 20;
+
+export type FictionPreviewType = {
```

Comment:
@cedrtang defined format is ComponentName[Props|States|Reducers]: FictionPreviewProps